src/arrayMethodReduce.js

```diff
@@ -5,7 +5,31 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let startFrom;
+    let accumulator;
+
+    // in the case when callback havent got
+    if (arguments.length < 1) {
```

Comment:
this is a hack
to avoid it, you need to have startFrom as `0` when there is no callback provided